model/src/main/kotlin/config/PackageConfiguration.kt

```diff
@@ -115,7 +115,7 @@ data class VcsMatcher(
                 "Matching against Git-Repo VCS info requires a non-blank path."
             }
         } else {
-            require(path == null) {
+            require(path.isNullOrBlank()) {
```

Comment:
Commit message: s/Accecpt/Accept/

Reply:
Is there any reason for this changes besides that it should be the opposite?

I believe specifying the VCS path is misleading for non Git Repo projects because the VCS path is not used for matching. E.g. specifying path="" to me indicates that only empty paths are matched.

Apart from that, we'd end up with two ways of specifying no VCS path.

Reply:
I removed the commit.